appveyor.yml

```diff
@@ -1,8 +1,8 @@
 environment:
   matrix:
-    - nodejs_version: '5'
-    - nodejs_version: '4'
-    - nodejs_version: '0.12'
```

Comment:
What's the rationale for dropping support for <6?

Reply:
- Align with Node's support policy by targeting oldest LTS.
- Use promise-based API out-of-the box (without having to use `bluebird`)
- Remove workarounds for old Node versions
- Refactor the internals to use ES syntax (scoped variables, spread args, arrow functions, etc.)

Reply:
I see. Looks that just this alone should be a different PR/discussion with the maintainers if they are aligned. That way, it becomes more progressive than big-bang.